extensions/vtk/src/OHIFVTKViewport.js

```diff
@@ -66,6 +68,7 @@ class OHIFVTKViewport extends Component {
     viewportIndex: PropTypes.number,
     children: PropTypes.node,
     onScroll: PropTypes.func,
+    servicesManager: PropTypes.object,
```

Comment:
Required? In general, prefer `required` as default unless there's a good reason to mark as optional. Optional means you have one more edge case to guard/test.